lib/yardstick/decorator.rb

```diff
@@ -0,0 +1,60 @@
+module Yardstick
+  # A string decorator for applying unix console codes
+  class Decorator
+    FORMAT = "\e[%{mode};%{color}%{string}\e[0m".freeze
+
+    COLOR_CODES = {
+      red:    31,
+      yellow: 33
+    }.freeze
+
+    MODE_CODES = {
+      bold:      1,
+      underline: 4
+    }.freeze
+
+    private_constant(:FORMAT, :COLOR_CODES, :MODE_CODES)
```

Comment:
Whoops I meant to say in the comment that I was using `private_constant(*constants(false))` which is what created the YARD issue